.github/workflows/Cabal.yml

```
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false  # Windows builds randomly fail
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        ghc: ["8.10.7", "9.2.2", "9.4.1"]
+        ghc: ["8.10.7", "9.2.2"]
```

The issue seems to be on Windows only so let's remove only that one:

```suggestion
        ghc: ["8.10.7", "9.2.2", "9.4.1"]
        exclude:
        - os: windows-latest
          ghc: 9.2.2
```

---

Ah, yes. Only WIndows' test fails. I've fixed it.